src/_image_wrapper.cpp

```diff
@@ -143,36 +144,52 @@ image_resample(PyObject *self, PyObject* args, PyObject *kwargs)
 
     if (!PyArg_ParseTupleAndKeywords(
             args, kwargs, "OOO|iO&dO&d:resample", (char **)kwlist,
-            &py_input_array, &py_output_array, &py_transform,
+            &py_input, &py_output, &py_transform,
             &params.interpolation, &convert_bool, &params.resample,
             &params.alpha, &convert_bool, &params.norm, &params.radius)) {
         return NULL;
     }
 
     if (params.interpolation < 0 || params.interpolation >= _n_interpolation) {
-        PyErr_Format(PyExc_ValueError, "invalid interpolation value %d",
+        PyErr_Format(PyExc_ValueError, "INvalid interpolation value %d",
```

Comment:
```suggestion
        PyErr_Format(PyExc_ValueError, "Invalid interpolation value %d",
```
(or was this on purpose?)

Reply:
oops, fixed.